jobs/haproxy/templates/haproxy.config.erb

```diff
@@ -170,7 +170,7 @@ frontend https-in
     <% if p("ha_proxy.accept_proxy") %>
     bind <%= p("ha_proxy.binding_ip") %>:443 accept-proxy ssl crt /var/vcap/jobs/haproxy/config/ssl
     <% else %>
-    bind <%= p("ha_proxy.binding_ip") %>:443 ssl crt /var/vcap/jobs/haproxy/config/ssl
+    bind <%= p("ha_proxy.binding_ip") %>:443 ssl crt /var/vcap/jobs/haproxy/config/ssl <% if p("ha_proxy.client_cert") == true %> ca-file <%= p("ha_proxy.client_cert_ca") %> verify optional crt-ignore-err all<% end %>
```

Comment:
Can we not ignore all cert errors by default? Seems like we would want to be aware of things like a bad CA, or expired cert.

Can this support CRLs so we can check against revoked client certs as well?